launchpod/angular-launchpod/backend/src/main/java/com/google/launchpod/data/LoginStatus.java

```diff
@@ -14,13 +14,32 @@
 
 package com.google.launchpod.data;
 
+import java.util.ArrayList;
+
 /** An item on a comment list. */
 public final class LoginStatus {
-  public final boolean isLoggedIn;
-  public final String message;
+  public boolean isLoggedIn;
+  public String message;
+  public ArrayList<UserFeed> feeds;
+
+  public static final String USER_FEED_KEY = "UserFeed";
+  public static final String TITLE_KEY = "title";
+  public static final String DESCRIPTION_KEY = "description";
+  public static final String NAME_KEY = "name";
+  public static final String EMAIL_KEY = "email";
+  public static final String XML_STRING_KEY = "xmlString";
+  public static final String POST_TIME_KEY = "postTime";
+  public static final String TIMESTAMP_KEY = "timestamp";
+
+  public void forSuccessfulLogin(String message, ArrayList<UserFeed> feeds) {
+    this.isLoggedIn = true;
+    this.message = message;
+    this.feeds = feeds;
+  }
 
```

Comment:
public static LoginStatus forSuccessfulLogin(String message, ArrayList<UserFeed> feeds) {
  return new LogicStatus(true, message, feeds);
}

private LoginStatus(isLoggedIn, message, Feeds) {
    this.isLoggedIn = isLoggedIn;
    this.message = message, ;
    this.feeds = feeds;
}

and remove all other constructors, so there are no public construtors

Reply:
done.